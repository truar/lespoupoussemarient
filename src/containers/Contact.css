.Contact {
    background: url('../images/contact.jpg') no-repeat center;
    background-size: cover;
    height: 101vh;  
    color: black;
    
    overflow: hidden;
}

.Contact h1 {
    display: none;
}

.Contact>.WrapperContent {
    align-items: flex-start;
    margin-left: 3%;
    margin-top: 10%;
    z-index: 10;
    text-shadow: none;  
}

.Contact ul {
    list-style: none;
    padding-left: 5%;
}

.Contact p {
    margin-top: 5%;
}

@media(min-width: 768px) {
    .Contact>.WrapperContent {
        margin-top: 20%;
        margin-left: 15%;
    }
    .Contact p {
        font-size: 1.75rem;
    }
    .Contact ul {
        font-size: 1.5rem;
    }
}

@media(min-width: 992px) {
    .Contact {
        align-items: center;
    }
    .Contact h1 {
        display: block;
    }
    .Contact>.WrapperContent {
        margin-top: 3%;
        margin-left: 0%;
        margin-left: 90px;
    }
}