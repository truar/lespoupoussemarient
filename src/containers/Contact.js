import React, {Component} from 'react';
import PageTitle from '../components/PageTitle';
import Section from '../components/Section';
import WrapperContent from '../components/WrapperContent';
import './Contact.css';

class Contact extends Component {
    render() {
        return (
            <Section id="Contact" className="Contact" {...this.props}>
                <PageTitle title="Coordonnées" />
                <WrapperContent id="coordonnees">
                    <p><span>Anouk POITEVIN :</span></p>
                    <ul>
                        <li className="mail">anouk.poitevin@gmail.com</li>
                        <li className="phone">06.85.28.45.72</li>
                    </ul>
                    
                    <p><span>Thibault RUARO :</span></p>
                    <ul>
                        <li className="mail">thibault.ruaro@gmail.com</li>
                        <li className="phone">06.25.56.27.53</li>
                    </ul>
                    <p><span>Adresse :</span></p>
                    <ul>
                        <li className="address">164 rue des Pyrénées<br/>75020 Paris</li>
                    </ul>
                </WrapperContent>
            </Section>
        );
    }
}

export default Contact;