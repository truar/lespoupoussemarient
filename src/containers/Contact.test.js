import React from 'react';
import Contact from './Contact';
import { shallow } from 'enzyme';

it('renders without crashing', () => {
  const component = shallow(<Contact />);
  expect(component).toMatchSnapshot();
});

