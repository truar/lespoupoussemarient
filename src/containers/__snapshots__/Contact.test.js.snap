// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Contact />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <PageTitle
          title="Coordonnées"
        />,
        <WrapperContent
          id="coordonnees"
        >
          <p>
            <span>
              Anouk POITEVIN :
            </span>
          </p>
          <ul>
            <li
              className="mail"
            >
              anouk.poitevin@gmail.com
            </li>
            <li
              className="phone"
            >
              06.85.28.45.72
            </li>
          </ul>
          <p>
            <span>
              Thibault RUARO :
            </span>
          </p>
          <ul>
            <li
              className="mail"
            >
              thibault.ruaro@gmail.com
            </li>
            <li
              className="phone"
            >
              06.25.56.27.53
            </li>
          </ul>
          <p>
            <span>
              Adresse :
            </span>
          </p>
          <ul>
            <li
              className="address"
            >
              164 rue des Pyrénées
              <br />
              75020 Paris
            </li>
          </ul>
        </WrapperContent>,
      ],
      "className": "Contact",
      "id": "Contact",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "title": "Coordonnées",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": Array [
            <p>
              <span>
                Anouk POITEVIN :
              </span>
            </p>,
            <ul>
              <li
                className="mail"
              >
                anouk.poitevin@gmail.com
              </li>
              <li
                className="phone"
              >
                06.85.28.45.72
              </li>
            </ul>,
            <p>
              <span>
                Thibault RUARO :
              </span>
            </p>,
            <ul>
              <li
                className="mail"
              >
                thibault.ruaro@gmail.com
              </li>
              <li
                className="phone"
              >
                06.25.56.27.53
              </li>
            </ul>,
            <p>
              <span>
                Adresse :
              </span>
            </p>,
            <ul>
              <li
                className="address"
              >
                164 rue des Pyrénées
                <br />
                75020 Paris
              </li>
            </ul>,
          ],
          "id": "coordonnees",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <span>
                Anouk POITEVIN :
              </span>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Anouk POITEVIN :",
              },
              "ref": null,
              "rendered": "Anouk POITEVIN :",
              "type": "span",
            },
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <li
                  className="mail"
                >
                  anouk.poitevin@gmail.com
                </li>,
                <li
                  className="phone"
                >
                  06.85.28.45.72
                </li>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "anouk.poitevin@gmail.com",
                  "className": "mail",
                },
                "ref": null,
                "rendered": "anouk.poitevin@gmail.com",
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "06.85.28.45.72",
                  "className": "phone",
                },
                "ref": null,
                "rendered": "06.85.28.45.72",
                "type": "li",
              },
            ],
            "type": "ul",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <span>
                Thibault RUARO :
              </span>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Thibault RUARO :",
              },
              "ref": null,
              "rendered": "Thibault RUARO :",
              "type": "span",
            },
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <li
                  className="mail"
                >
                  thibault.ruaro@gmail.com
                </li>,
                <li
                  className="phone"
                >
                  06.25.56.27.53
                </li>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "thibault.ruaro@gmail.com",
                  "className": "mail",
                },
                "ref": null,
                "rendered": "thibault.ruaro@gmail.com",
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "06.25.56.27.53",
                  "className": "phone",
                },
                "ref": null,
                "rendered": "06.25.56.27.53",
                "type": "li",
              },
            ],
            "type": "ul",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <span>
                Adresse :
              </span>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Adresse :",
              },
              "ref": null,
              "rendered": "Adresse :",
              "type": "span",
            },
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <li
                className="address"
              >
                164 rue des Pyrénées
                <br />
                75020 Paris
              </li>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "164 rue des Pyrénées",
                  <br />,
                  "75020 Paris",
                ],
                "className": "address",
              },
              "ref": null,
              "rendered": Array [
                "164 rue des Pyrénées",
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {},
                  "ref": null,
                  "rendered": null,
                  "type": "br",
                },
                "75020 Paris",
              ],
              "type": "li",
            },
            "type": "ul",
          },
        ],
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <PageTitle
            title="Coordonnées"
          />,
          <WrapperContent
            id="coordonnees"
          >
            <p>
              <span>
                Anouk POITEVIN :
              </span>
            </p>
            <ul>
              <li
                className="mail"
              >
                anouk.poitevin@gmail.com
              </li>
              <li
                className="phone"
              >
                06.85.28.45.72
              </li>
            </ul>
            <p>
              <span>
                Thibault RUARO :
              </span>
            </p>
            <ul>
              <li
                className="mail"
              >
                thibault.ruaro@gmail.com
              </li>
              <li
                className="phone"
              >
                06.25.56.27.53
              </li>
            </ul>
            <p>
              <span>
                Adresse :
              </span>
            </p>
            <ul>
              <li
                className="address"
              >
                164 rue des Pyrénées
                <br />
                75020 Paris
              </li>
            </ul>
          </WrapperContent>,
        ],
        "className": "Contact",
        "id": "Contact",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "title": "Coordonnées",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": Array [
              <p>
                <span>
                  Anouk POITEVIN :
                </span>
              </p>,
              <ul>
                <li
                  className="mail"
                >
                  anouk.poitevin@gmail.com
                </li>
                <li
                  className="phone"
                >
                  06.85.28.45.72
                </li>
              </ul>,
              <p>
                <span>
                  Thibault RUARO :
                </span>
              </p>,
              <ul>
                <li
                  className="mail"
                >
                  thibault.ruaro@gmail.com
                </li>
                <li
                  className="phone"
                >
                  06.25.56.27.53
                </li>
              </ul>,
              <p>
                <span>
                  Adresse :
                </span>
              </p>,
              <ul>
                <li
                  className="address"
                >
                  164 rue des Pyrénées
                  <br />
                  75020 Paris
                </li>
              </ul>,
            ],
            "id": "coordonnees",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <span>
                  Anouk POITEVIN :
                </span>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Anouk POITEVIN :",
                },
                "ref": null,
                "rendered": "Anouk POITEVIN :",
                "type": "span",
              },
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <li
                    className="mail"
                  >
                    anouk.poitevin@gmail.com
                  </li>,
                  <li
                    className="phone"
                  >
                    06.85.28.45.72
                  </li>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "anouk.poitevin@gmail.com",
                    "className": "mail",
                  },
                  "ref": null,
                  "rendered": "anouk.poitevin@gmail.com",
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "06.85.28.45.72",
                    "className": "phone",
                  },
                  "ref": null,
                  "rendered": "06.85.28.45.72",
                  "type": "li",
                },
              ],
              "type": "ul",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <span>
                  Thibault RUARO :
                </span>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Thibault RUARO :",
                },
                "ref": null,
                "rendered": "Thibault RUARO :",
                "type": "span",
              },
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <li
                    className="mail"
                  >
                    thibault.ruaro@gmail.com
                  </li>,
                  <li
                    className="phone"
                  >
                    06.25.56.27.53
                  </li>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "thibault.ruaro@gmail.com",
                    "className": "mail",
                  },
                  "ref": null,
                  "rendered": "thibault.ruaro@gmail.com",
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "06.25.56.27.53",
                    "className": "phone",
                  },
                  "ref": null,
                  "rendered": "06.25.56.27.53",
                  "type": "li",
                },
              ],
              "type": "ul",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <span>
                  Adresse :
                </span>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Adresse :",
                },
                "ref": null,
                "rendered": "Adresse :",
                "type": "span",
              },
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <li
                  className="address"
                >
                  164 rue des Pyrénées
                  <br />
                  75020 Paris
                </li>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    "164 rue des Pyrénées",
                    <br />,
                    "75020 Paris",
                  ],
                  "className": "address",
                },
                "ref": null,
                "rendered": Array [
                  "164 rue des Pyrénées",
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {},
                    "ref": null,
                    "rendered": null,
                    "type": "br",
                  },
                  "75020 Paris",
                ],
                "type": "li",
              },
              "type": "ul",
            },
          ],
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
